aten/src/ATen/TensorImpl.cpp

```diff
@@ -46,6 +46,12 @@ IntList TensorImpl::sizes() const {
   return IntList(THTensor_getSizePtr(tensor), dim());
 }
 
+IntList TensorImpl::strides() const {
+  // NB: THTensor doesn't agree with Tensor for scalars, so we
```

Comment:
I think I made this comment on sizes but I don't remember the response: we always have to create a new IntList because the data type isn't an IntList right?  (Maybe you meant non-implicit IntList?)

Reply:
yeah, it looks like I updated one comment but not the other.